upgrade/cleanup.sh

```diff
@@ -2,27 +2,53 @@
 
 set -e
 
-kubectl get zfsvolumes.openebs.io -n openebs -oyaml > volumes.yaml
+if [ -z $1 ]; then
+	# default namespace is openebs when all the custom resources are created
+	ZFSPV_NAMESPACE="openebs"
+else
+	ZFSPV_NAMESPACE=$1
+fi
 
-# remove the finalizer from the old CR
-sed -i "/zfs.openebs.io\/finalizer/d" volumes.yaml
-kubectl apply -f volumes.yaml
+echo "Fetching ZFS Volumes"
+numVol=`kubectl get zfsvolumes.openebs.io --no-headers -n $ZFSPV_NAMESPACE | wc -l`
 
-# delete the old CR
-kubectl delete -f volumes.yaml
+if [ $numVol -gt 0 ]; then
+	echo "Cleaning the ZFS Volumes(`expr $numVol - 1`)"
```

Comment:
why $numVol - 1  ??

Reply:
earier I was not using --no-headers, which will give one extra count, so added -1 here. Now added  --no-headers but forgot to fix this message.  Thanks @w3aman .fixed it